doc/source/python/tree_attributes.rst

```diff
@@ -29,6 +29,7 @@ Tree attributes
     attribute_topological_height
     attribute_tree_sampling_probability
     attribute_volume
+	attribute_moment_of_inertia
```

Comment:
incorrect spacing + move the function name in the list to respect the alphabetical sort